app/src/componentLibrary/modal/modalLayout/modalLayout.scss

```diff
@@ -39,6 +39,7 @@ $Z-INDEX-OVERLAY: 2;
   width: 100%;
   height: 100%;
   text-align: center;
+  background-color: rgba($COLOR--e-400, 0.35);
```

Comment:
Component have `overlay` property. Please move background to .overlay-default class like .overlay-light-cyan and set `overlay: 'default'` in defaultProps